src/cijoe/core/command.py

```diff
@@ -268,7 +268,7 @@ def getconf(self, key: str, default: Any = None):
         envvar = os.getenv(envkey)
         if envvar:
             log.debug(f"found {key} ({envkey}) in environment variables.")
-            return envvar
+            return convert_str(envvar)
```

Comment:
This should probably be added to the doc-string. To me it is not entirely logical that this is the behaviour

Reply:
When you view ``getconf()`` as a method for retrieving values from configuration files with environment variable overrides, converting ``.toml`` values from strings to booleans or integers makes sense. This change ensures scalar values are treated the same, aligning with the configuration retrieval behavior, rather than returning unconverted values.